### ISSUE TITLE
Accuracy is significantly lower than 99.5%

### ISSUE THREAD

**Issue by tabVersion** [hello2all/GTSRB_Keras_STN#4]
Hello
Please check you uploaded the correct weight file
![image](https://user-images.githubusercontent.com/14017336/53282417-42524c80-3772-11e9-8c0b-2105337dfc5c.png)

**Comment by hello2all** [comment]
Thank you for your interest in this project. The uploaded weight file is correct and has been verified many times. You may use the provided `test_model.ipynb` notebook to view the testing accuracy. However, it has been quite a long time since completetion of this project and the newer versions of Tensorflow and Keras might have compatibility issues with older weight files. Could you:

1. Run the notebook and see if the testing accuracy is correct
2. If not, please let me know which version of Tensorflow and Keras you have been using.

The lastest correct results was tested on:
Tensorflow 1.3.0
Keras 2.0.8

**Comment by tabVersion** [comment]
Sorry for not checking the message box. I used a wrong way to test the results.
Your model is really awesome. I retrained the model and get 99% accuracy after 130 epochs.

**Comment by sisrfeng** [comment]
Hi, what versions of these packages should we use?
```
python
matplotlib
scipy
numpy
pandas
scikit-learn
```

thx!
